cli-plugins/apps/command-reference/clustersupplychain.hbs.md

```diff
@@ -0,0 +1,45 @@
+# Tanzu Apps Cluster Supply Chain
+
+## tanzu apps cluster supply chain list
+
+`tanzu apps clustersupplychain get` command lists the available supply chains installed in the cluster.
```

Comment:
oops . didn't catch this before. this is the wrong command isn't it?
It's supposed to be `tanzu apps clustersupplychain list` yeah? 

So maybe we should go with 
`tanzu apps clustersupplychain list` command lists the available supply chains installed in the cluster (supported `clustersupplychain` alias is `csc`).

Reply:
Sorry, I missed this too. Thanks :)